### ISSUE TITLE
`settings.json` linting for docker extension doesn't accept "Image" but sidebar does

### ISSUE THREAD

**Issue by Ushcode** [microsoft/vscode-docker#4325]
```json
"docker.containers.label": "Image",
"docker.containers.description": [
        "ContainerName",
        "Status",
        "Image"
    ]
```
Both flag warnings in the `settings.json` file, but adding the change correctly displays the Image name in the docker extension sidebar as expected

![image](https://github.com/microsoft/vscode-docker/assets/102725207/e0b8e09e-27ba-4b50-8010-1df7539f71cd)

![image](https://github.com/microsoft/vscode-docker/assets/102725207/a04d24c0-02b8-44b1-a377-837014faa274)

![image](https://github.com/microsoft/vscode-docker/assets/102725207/d8fde67a-d751-4758-99e8-9c288601a487)

Warning returned is:

`Value is not accepted. Valid values: "Compose Project Name", "ContainerId", "ContainerName", "CreatedTime", "FullTag", "ImageId", "Networks", "Ports", "Registry", "Repository", "RepositoryName", "RepositoryNameAndTag", "State", "Status", "Tag".(1)`

**Comment by bwateratmsft** [comment]
I am able to reproduce.

**Comment by bwateratmsft** [comment]
This is now released in Docker extension version 1.29.4.
